mention-bot.js

```diff
@@ -301,7 +328,8 @@ function guessOwners(
     .filter(function(owner) {
       return config.userBlacklist.indexOf(owner) === -1;
     })
-    .slice(0, 3);
+    .slice(0, config.maxReviewers)
+    .concat(defaultOwners);
```

Comment:
yeah, this should absolutely bypass config.maxReviewers. Here's the best thing i can think of:

- pick the best 3
- add the defaultOwners only if they don't appear in the best 3 already.